index.js

```
@@ -86,100 +92,204 @@ const getPlaylistItems = async (token, playlistUrl) => {
+
+    console.log("Getting spotify tracks.");
     const tracks = await getPlaylistItems(token, "https://open.spotify.com/playlist/28oszO2MY6o97B3yYFkiWO?si=6c6496aa66f842d7&pt=a0e5e4e29b041ec052bc045b00afc2d7");
-    const youtubeVideoUrls = [];
 
-    for (const track of tracks){
-        youtubeVideoUrls.push(await getYoutubeSongUrl(track.name, track.artists));
-    }
-    
-    
+    console.log("Getting youtube urls");
+    tracks = await getYoutubeSongUrls(tracks);
```

Biome

[error] Can't assign tracks because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)